plugins/hydra_ray_launcher/example/local/config.yaml

```diff
@@ -0,0 +1,4 @@
+defaults:
+  - hydra/launcher: ray_local
+
+task: 1
```

Comment:
since we are moving to a more meaningful example, maybe we can change task too. (random_seed seems okay, or something about configuring the training or the model).